Wikipedia/Code/EmbeddableView.swift

```diff
@@ -0,0 +1,9 @@
+/* Makes it easier to use custom xib view classes in another xib or storyboard. */
+class EmbeddableView: UIView {
+    override func awakeAfter(using aDecoder: NSCoder) -> Any? {
```

Comment:
This is more confusing and prone to break than using one of the built-in storyboard/xib composition methods. I'd suggest embedding child view controllers or just having duplicate labels and switches for the edit option views.